kt/godot-runtime/src/main/kotlin/godot/annotation/CoreTypeHelper.kt

```diff
@@ -0,0 +1,10 @@
+package godot.annotation
+
+/**
+ * Only applicable to godot core type helper functions generated in the api gen
+ *
+ * Here for the IDE checks to differentiate mutation function calls (these helper functions) from regular function calls
+ */
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class CoreTypeHelper
```

Comment:
make this internal? I don't think this will be used by user code.

Reply:
Not possible. The generated code is in `godot-library` but the annotation is in `godot-runtime` because of the core types. I cannot have it internal. Then it's not visible in `godot-library` 🙈